Runtime/World/CFluidPlaneGPU.cpp

```diff
@@ -52,19 +52,19 @@ void CFluidPlaneGPU::RenderStripWithRipples(float curY, const CFluidPlaneRender:
           break;
       }
 
-      std::fill(std::begin(pv.m_outerLevels), std::end(pv.m_outerLevels), subdivF);
-      std::fill(std::begin(pv.m_innerLevels), std::end(pv.m_innerLevels), subdivF);
+      pv.m_outerLevels.fill(subdivF);
+      pv.m_innerLevels.fill(subdivF);
     } else {
-      bool r19 = (flags[yTile + 1][xTile] & 0x2) != 0; // North
-      bool r16 = (flags[yTile][xTile - 1] & 0x8) != 0; // West
-      bool r18 = (flags[yTile][xTile + 1] & 0x4) != 0; // East
-      bool r17 = (flags[yTile - 1][xTile] & 0x1) != 0; // South
+      const bool r19 = (flags[yTile + 1][xTile] & 0x2) != 0; // North
+      const bool r16 = (flags[yTile][xTile - 1] & 0x8) != 0; // West
+      const bool r18 = (flags[yTile][xTile + 1] & 0x4) != 0; // East
+      const bool r17 = (flags[yTile - 1][xTile] & 0x1) != 0; // South
```

Comment:
Probably be a good idea to go ahead and rename these `r#` variables something meaningful

Reply:
Done. I've put the changes in a separate commit to keep it separate from the std::array conversion changes.

Reply:
LGTM